src/app/externs/backendapi.js

```diff
@@ -1147,6 +1147,51 @@ backendApi.PersistentVolumeClaimList;
 
 /**
  * @typedef {{
+ *   objectMeta: !backendApi.ObjectMeta,
+ *   typeMeta: !backendApi.TypeMeta,
+ *   scaleTargetRef: !backendApi.ScaleTargetRef,
+ *   minReplicas: !number,
```

Comment:
Why is some `!number` and some `number` here? They are actually the same types. Primitve types are non-nullable by default. Nice reading at: https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler

Same for strings below.